docs/docs/reference/dropped-features/package-objects.md

```diff
@@ -42,3 +42,6 @@ in a source file `src.scala`, it could be invoked from the command line using a
 "program name" is mangled it is recommended to always put `main` methods in explicitly named objects.
 
 **Note 3:** The notion of `private` is independent of whether a definition is wrapped or not. A `private` toplevel definition is always visible from everywhere in the enclosing package.
+
+**Note 4:** If several wrapper objects contain toplevel definitions with the same name,
```

Comment:
It looks like this is true only for separate compilation. If you have several files, each with an overloaded version of a method, and compile them in one run, the overload will work. If you compile them separately, only the last compiled method will be visible.